ceremony/2022-09-27/staged/rekor.json

```diff
@@ -0,0 +1,10 @@
+{
+	"signed": {
+		"_type": "targets",
+		"spec_version": "1.0",
+		"version": 4,
+		"expires": "2023-03-27T19:57:02Z",
+		"targets": {}
+	},
+	"signatures": null
+}
```

Comment:
Is this correct, to have no signatures to mark a delegation as no longer used? Or should we just not include the delegated metadata?

Reply:
Given this isn't referenced in the metadata, i assume deleted?

Reply:
It is deleted! i think go-tuf never gets rid of the file in the store: you still are required to forever persist delegation versions (to prevent rollbacks), and I think that's the reasoning @mnm678 @trishankatdatadog

Reply:
Not necessarily. You can use the root to rotate the timestamp and/or snapshot keys, which will require clients to reset any previous snapshot metadata (see [5.3.11](https://theupdateframework.github.io/specification/v1.0.31/index.html#update-root)).